chanutils/concurrency_test.go

```diff
@@ -0,0 +1,55 @@
+package chanutils
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestErrGroup(t *testing.T) {
+	errs := []error{errors.New("error #1"), errors.New("error #2")}
+
+	t.Parallel()
```

Comment:
Swap the ordering here to do this first before creating the error slice? 

Alternatively, the error slice can be a private package level variable.

Reply:
swapped

but it doesn't matter btw, because parallel execution will be only in `t.Run`

I agree with @positiveblue to keep this var inside func and not as package level variable (https://github.com/lightninglabs/taro/pull/186#discussion_r1015212659).